src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -781,11 +787,37 @@ function loadOperation(
             IsContentType: isContentType,
             IsEndpoint: false,
             SkipUrlEncoding: false, //TODO: retrieve out value from extension
-            Explode: false,
-            Kind: kind
+            Explode:
+                (inputType as InputListType).ElementType &&
+                format &&
+                format === "multi"
+                    ? true
+                    : false,
```

Comment:
looks like this condition is just: `format === "multi" && inputType is InputListType`.
Could we simplify this?
As far as I know, if `format` is null or undefined, `format === "multi" will be false, therefore the check for `format` itself is redundant.

Reply:
We cannot use `inputType is InputListType` because of build failure. I removed `format` check to simplify.